WalletWasabi.Fluent/CrashReport/Views/CrashReportWindow.axaml

```diff
@@ -0,0 +1,47 @@
+<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:vm="using:WalletWasabi.Fluent.CrashReport.ViewModels"
+        xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
+        x:Class="WalletWasabi.Fluent.CrashReport.Views.CrashReportWindow"
+        Width="640" Height="480"
+        MinWidth="640" MinHeight="480"
+        MaxWidth="640" MaxHeight="480"
+        Title="{Binding Title}"
+        WindowStartupLocation="CenterScreen"
+        ExtendClientAreaToDecorationsHint="True"
+        ExtendClientAreaChromeHints="PreferSystemChrome"
+        x:Name="CrashReporterWindow"
+        x:CompileBindings="True"
+        x:DataType="vm:CrashReportWindowViewModel">
+  <Panel Margin="{Binding #CrashReporterWindow.OffScreenMargin}">
+    <Viewbox Width="300" Height="300" Opacity="0.1" VerticalAlignment="Center"
+             HorizontalAlignment="Right">
+      <ContentControl Template="{DynamicResource wasabi_logo_dynamic}" />
+    </Viewbox>
+    <controls:ContentArea EnableNext="True"
+                          NextContent="Exit"
+                          EnableCancel="{Binding !HideTraceTextBox}"
+                          CancelContent="Restart Wasabi"
+                          FocusNext="True"
+                          Background="Transparent"
+                          Title="{Binding Title}"
+                          Caption="{Binding Caption}"
+                          Margin="0 50 0 0"
+                          ScrollViewer.HorizontalScrollBarVisibility="Disabled"
+                          ScrollViewer.VerticalScrollBarVisibility="Disabled">
```

Comment:
I would remove the
```
Background="Transparent"
Margin="0 50 0 0"
```

And swap the `ViewBox` and `ContentArea` order and add an `IsHitTestVisible=false` to the viewbox.

and add this:
```
<controls:ContentArea.Styles>
  <Style Selector="controls|ContentArea /template/ DockPanel#MainDockPanel">
    <Setter Property="Margin" Value="0 36 0 0" />
  </Style>
</controls:ContentArea.Styles>
```

We should use colors that actually represent the new generation of Wasabi.
So basically what is changed is the background color and top margin size.

Whole XAML:
```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="using:WalletWasabi.Fluent.CrashReport.ViewModels"
        xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
        x:Class="WalletWasabi.Fluent.CrashReport.Views.CrashReportWindow"
        Width="640" Height="480"
        MinWidth="640" MinHeight="480"
        MaxWidth="640" MaxHeight="480"
        Title="{Binding Title}"
        WindowStartupLocation="CenterScreen"
        ExtendClientAreaToDecorationsHint="True"
        ExtendClientAreaChromeHints="PreferSystemChrome"
        x:Name="CrashReporterWindow"
        x:CompileBindings="True"
        x:DataType="vm:CrashReportWindowViewModel">
  <Panel Margin="{Binding #CrashReporterWindow.OffScreenMargin}">
    <controls:ContentArea EnableNext="True"
                          NextContent="Exit"
                          EnableCancel="{Binding !HideTraceTextBox}"
                          CancelContent="Restart Wasabi"
                          FocusNext="True"
                          Title="{Binding Title}"
                          Caption="{Binding Caption}"
                          ScrollViewer.HorizontalScrollBarVisibility="Disabled"
                          ScrollViewer.VerticalScrollBarVisibility="Disabled">
      <controls:ContentArea.Styles>
        <Style Selector="controls|ContentArea /template/ DockPanel#MainDockPanel">
          <Setter Property="Margin" Value="0 36 0 0" />
        </Style>
      </controls:ContentArea.Styles>
      <DockPanel LastChildFill="True">
        <controls:InfoMessage DockPanel.Dock="Bottom" VerticalAlignment="Center" HorizontalAlignment="Center"
                              Margin="0 20 0 0">
          <WrapPanel>
            <TextBlock Text="{Binding SuggestionTextFragment1}" />
            <Button Margin="5 0" Classes="activeHyperLink" Content="GitHub repo here"
                    Command="{Binding OpenGitHubRepoCommand}" />
            <TextBlock Text="{Binding SuggestionTextFragment2}" />
          </WrapPanel>
        </controls:InfoMessage>
        <TextBox IsVisible="{Binding !HideTraceTextBox}" Classes="selectableTextBlock monoSpaced wrap"
                 Text="{Binding Trace}" />
      </DockPanel>
    </controls:ContentArea>
    <Viewbox Width="300" Height="300" Opacity="0.1" VerticalAlignment="Center"
             HorizontalAlignment="Right" IsHitTestVisible="False">
      <ContentControl Template="{DynamicResource wasabi_logo_dynamic}" />
    </Viewbox>
  </Panel>
</Window>
```

![image](https://user-images.githubusercontent.com/16364053/129334371-c25179f6-28fb-40e9-bfab-5fbb034ef582.png)